zerver/webhooks/github/view.py

```diff
@@ -482,17 +480,14 @@ def get_pull_request_review_requested_body(helper: Helper) -> str:
 
     all_reviewers = []
 
-    for reviewer in requested_reviewers:
+    for reviewer in requested_reviewer:
         all_reviewers.append("[{login}]({html_url})".format(**reviewer))
 
-    for team_reviewer in requested_team_reviewers:
+    for team_reviewer in requested_team:
         all_reviewers.append("[{name}]({html_url})".format(**team_reviewer))
 
     reviewers = ""
-    if len(all_reviewers) == 1:
-        reviewers = all_reviewers[0]
-    else:
-        reviewers = "{} and {}".format(", ".join(all_reviewers[:-1]), all_reviewers[-1])
+    reviewers = all_reviewers[0]
```

Comment:
Is this hunk correct?  WIth the append logic above, it seems like we could have a user and a team.

Reply:
I think this should be fine, though I don't find the `requested_team` field in the [API docs](https://docs.github.com/en/developers/webhooks-and-events/webhook-events-and-payloads#pull_request), I assume that there will be either `requested_reviewer` or `requested_team`, as that is specific to the review request. The `requested_reviewers` and `requested_teams` come in as an addon with all requested people.

Reply:
@timabbott its correct because we only support either 1 team or 1 reviewer per message and this code does that.

Reply:
@adambirds do you have something that talks about `requested_team` didn't find anything.

Reply:
@LoopThrough-i-j its not documented, but its only available on Github Team or above plans. We have the fixture from when this was first added and you get a `requested_team` field instead of `requested_reviewer`.